tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -346,6 +341,12 @@ private ChunkHeader readChunkHeader(long position, boolean markerRead) throws IO
    * @return the pages of this chunk
    */
   public ByteBuffer readChunk(ChunkHeader header) throws IOException {
+    if (this.readVersionNumber().startsWith("v")) {
+      config.setEndian("LITTLE_ENDIAN");
+    }
+    else {
+      config.setEndian("BIG_ENDIAN");
+    }
```

Comment:
I think this will cause a mess if two files of different endian are being read concurrently.

Reply:
> I think this will cause a mess if two files of different endian are being read concurrently.

Yes, it may cause a mess, but I have no other idea about how and where to determine the endian type.

Reply:
You may add a field in TsFileSequenceReader to indicate the byte order and pass it down throughout the query process.